plugin/cache.go

```diff
@@ -74,6 +74,22 @@ func opKeysRegex(path string) *regexp.Regexp {
 	return regexp.MustCompile(expr)
 }
 
+// opKeyRegex returns a regex that matches the key <op>::<path>.
+// It is useful for clearing a specific cached op for the entry
+// corresponding to <path>.
+func opKeyRegex(op string, path string) *regexp.Regexp {
```

Comment:
What changed here?

Reply:
I want to clear out `List::<parent>` only in `plugin.Delete`. `opKeysRegex` clears out `Op::<parent>` and `Op::<children>`.